packages/react/src/components/layout/GridLayout.tsx

```diff
@@ -25,30 +25,35 @@ export interface GridLayoutProps extends React.HTMLAttributes<HTMLDivElement> {
  * <LiveKitRoom>
  * ```
  */
-export function GridLayout({ filter, filterDependencies, ...props }: GridLayoutProps) {
-  const participants = useParticipants();
-  const filterDependenciesArray = filterDependencies ?? [];
-  const filteredParticipants = React.useMemo(() => {
-    return filter ? participants.filter(filter) : participants;
-  }, [filter, participants, ...filterDependenciesArray]);
+export function GridLayout({ filter, filterDependencies = [], ...props }: GridLayoutProps) {
+  const participantOptions: UseParticipantsOptions = React.useMemo(() => {
+    return { updateOnlyOn: [] };
+  }, []);
+
+  const participants = useParticipants(participantOptions);
```

Comment:
What is the reason for using useMemo? Could this be replaced with: 
`const participants = useParticipants({ updateOnlyOn: [] });`

Reply:
left over from debugging, fixed